src/Tribe/Attendee_Repository.php

```diff
@@ -43,7 +43,9 @@ class Tribe__Tickets__Attendee_Repository extends Tribe__Repository {
 	public function __construct() {
 		parent::__construct();
 
-		$this->create_args['post_type'] = current( $this->attendee_types() );
+		$this->create_args['post_type']  = current( $this->attendee_types() );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 3 spaces but found 2 spaces

<sub>hash: fd2827eb94b133e200b07e06ecea98ef</sub>